docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,161 @@
+You just write your code within these functions, and TestCafe executes the code on the client, thus you will obtain the actual state of the page.
+
+For example, clicking the Submit button on the sample web page opens a "Thank you" page.
+To get access to DOM elements on the opened page, the `Selector` function can be used.
+The following example demonstrates how to access the article header element and obtain its actual text.
+
+```js
+import { Selector } from 'testcafe';
+
+// Declare the parameterized Selector function
+// to get access to a DOM element identified by the `id` attribute
+const getElementById = Selector((id) => document.querySelector(`#${id}`));
+
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('Test1', async t => {
+    await t
+        .typeText('#developer-name', 'John Smith')
+        .click('#submit-button');
+
+    // Use the Selector function to get access to the article header
+    const articleHeader = await getElementById('article-header');
+
+    // Obtain the text of the article header
+    const headerText = articleHeader.textContent;
+});
+```
+
+For more information, see [Executing Client Code](../test-api/executing-client-code.md).
+
+### Assertions
+
+A functional test also should check the result of actions performed.
+For example, the article header on the "Thank you" page should address a user by the entered name.
+To check if the header is correct, you have to add an assertion to the test.
+
+You can use assertions from Node's built-in [assert](https://nodejs.org/api/assert.html) module or from any third-party assertion library.
+Before calling assertions, make sure an assertion library is installed into your project.
+
+The following test demonstrates how to use an assertion from [Chai Assertion Library](http://chaijs.com/api/bdd/).
+Before running the test, install the assertion library by calling the `npm install -g chai` command.
+
+```js
+import { expect } from 'chai';
+import { Selector } from 'testcafe';
+
+// Declare the parameterized hybrid function
+// to obtain text content of an element identified by the `id` attribute
+const getElementById = Selector((id) => document.querySelector(`#${id}`));
+
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('Test1', async t => {
+    await t
+        .typeText('#developer-name', 'John Smith')
+        .click('#submit-button');
+
+    // Use the Selector function to get access to the article header
+    const articleHeader = await getElementById('article-header');
+
+    // Obtain the text of the article header
+    const headerText = articleHeader.textContent;
+
+    // Use the assertion to check if the actual text is equal to the expected one
+    expect(headerText).to.equal('Thank you, John Smith!');
```

Comment:
We can go without `headerText` intermediate var now. BTW, I think `innerText` suits better here then `textContent` (I'm not sure if `textContent` will be actually equal to `Thank you, John Smith!`).